.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## What is the goal of this pull request?
+
+## What does this pull request change?
+
+## What work remains to be done?
+
+## Does this pull request change the functionality of the code?
+
+YES | NO
+
+### How?
+
+## Do you consider it adequately tested?
+
+YES | NO
+
+## Does it change the deployment pipeline?
```

Comment:
I think we can probably get rid of this question since there's nothing being deployed (other than perhaps the website changes, but maybe we can have a different template in the gh-pages branch? doesn't that even work?).